docs/wmco-prerequisites.md

```diff
@@ -2,8 +2,15 @@
 Below are the Cluster and OS prerequisites for WMCO. Please also see the vSphere-specific section that 
 may be relevant.
 
-## Supported Cloud Providers based on OKD/OCP Version and WMCO version
-| Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
+## Supported Installation method
+* Installer-Provisioned Infrastructure (IPI) installation method is the recommended and supported installation method.
+
+* User-Provisioned Infrastructure (UPI) is **only supported** on Bare metal or Provider-agnostic
+  platform (`platform: none`) for the [BYOH (Bring Your Own Host)](byoh-instance-pre-requisites.md#byoh-instance-pre-requisites)
+  use case. UPI is not supported in any other platform.
```

Comment:
Maybe I am being stupid here. This refers to the initial install of OCP, not Windows nodes, right?  We do not support Windows nodes being installed on UPI, except for BYOH on